lib/scenic_font_press_start_2p.ex

```diff
@@ -1,18 +1,17 @@
 defmodule ScenicFontPressStart2p do
-  @hash Application.app_dir(:scenic_font_press_start_2p, ["priv", "PressStart2P.ttf"])
-        |> Scenic.Cache.Hash.file!(:sha)
+  @font_hash "eLcVQnSbL2kg9g_ix0_oVzgoTcuc0mp_9qZCxU04Rws"
+
+  @font_metrics :code.priv_dir(:scenic_font_press_start_2p) |> Path.join("PressStart2P.ttf.metrics")
+  @font_metrics_hash Scenic.Cache.Support.Hash.file!(@font_metrics, :sha)
+  @font_folder :code.priv_dir(:scenic_font_press_start_2p) |> to_string()
```

Comment:
I think you want the `:code.priv_dir` calls to be done at runtime since the priv dir changes directories between compile and run time.

Reply:
Yup, that solves the case of the mysterious font not working on device issue. ❤️